build/presubmit.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+
+./build/check_gofmt.sh .
```

Comment:
Ack. An advantage with make is that you can remember a few build rule names
and not have to type the entire build, test and vet commands. But as you
say its not a big deal.

On Wed, May 20, 2015 at 11:01 AM, Victor Marmol <notifications@github.com>
wrote:

> In build/presubmit.sh
> <https://github.com/google/cadvisor/pull/722#discussion_r30730446>:
>
> > +# Licensed under the Apache License, Version 2.0 (the "License");
> > +# you may not use this file except in compliance with the License.
> > +# You may obtain a copy of the License at
> > +#
> > +#     http://www.apache.org/licenses/LICENSE-2.0
> > +#
> > +# Unless required by applicable law or agreed to in writing, software
> > +# distributed under the License is distributed on an "AS IS" BASIS,
> > +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +# See the License for the specific language governing permissions and
> > +# limitations under the License.
> > +
> > +set -e
> > +set -x
> > +
> > +./build/check_gofmt.sh .
>
> If our flow becomes more complex (like Heapster's build), then I can't
> complain about moving to make :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/722/files#r30730446>.
>